typescript/infra/scripts/core.ts

```diff
@@ -14,7 +14,7 @@ async function main() {
   const multiProvider = await config.getMultiProvider();
   const deployer = new AbacusCoreDeployer(
     multiProvider,
-    config.core.validatorManagers,
```

Comment:
was this a mistake before? if so wondering how ts didn't catch it

Reply:
Yeah, config is currently typed as `any`